src/_components/NavBarComponent.jsx

```diff
@@ -14,8 +14,8 @@ export class NavBarComponent extends Component {
 
   render() {
     const { title } = this.props;
-    const token = jwt.decode(localStorage.getItem('token'));
-    const { username = '' } = token || {};
+    const token = jwt.decode(localStorage.getItem('art-prod-web-token'));
+    const { name, picture } = token || {};
```

Comment:
What happens when the token is undefined? `name` and `picture` will be undefined. Will this affect the UI?